apps/epic-react/schemas/index.ts

```diff
@@ -0,0 +1,72 @@
+// —— documents
+import cta from './documents/cta'
+import exercise from './documents/exercise'
+import explainer from './documents/explainer'
+import interview from './documents/interview'
```

Comment:
It looks like the interview document doesn't actually exist

Reply:
Thanks for catching this, updated the PR to include that file.